adoc/admin-security-user-group-management.adoc

```diff
@@ -64,6 +64,7 @@ ldapmodify -v -H <LDAP_PROTOCOL>://<LDAP_NODE_FQDN><LDAP_NODE_PROTOCOL> -D "<BIN
 dn: cn=GROUP,ou=OU_NAME,dc=example,dc=org
 changetype: add
 objectClass: top
+objectClass: posixGroup
```

Comment:
Why do we need posixGroup? How is this related to the CaaSP DEX configuration?

Reply:
Our examples use `gidNumber`.  Without it, user will encounter errors as following:

seem to need to add : objectclass: posixGroup as following to avoid
ldap_add: Object class violation (65)
additional info: attribute "gidNumber" not allowed

Reply:
So we should remove the gidNumber and not add posixGroup IMO.
(the whole posix attributes are not required or useful when using LDAP just for DEX)

Reply:
hmm, it seems making sense to me if they are meaningless in our case.

Reply:
@cclhsu Can we remove gidNumber? This comment thread is the only thing blocking the merge.

Reply:
removed.